lib/ansible/modules/cloud/cloudstack/cs_image_store.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2019, Patryk Cichy @PatTheSilent
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: cs_image_store
+
+short_description: Manages CloudStack Image Stores.
+
+version_added: "2.8"
+
+description:
+    - "Deploy, remove, recreate CloudStack Image Stores."
+
+options:
+    url:
+        description:
+            - The URL for the Image Store.
+        required: true
```

Comment:
url is not requried according the arg_spec

Reply:
Ah, I've added `required_if` in the `AnsibleModule` initialization and forgot to remove that from there.
BTW shouldn't sanity checks catch something like that?

Reply:
good point, i would have expected that yes.

Reply:
Corrected.